code/modules/mob/living/simple_animal/abnormality/_tools/zayin/wishwell.dm

```diff
@@ -128,123 +125,121 @@
+		if(locate(I) in tethitem)
+			output = "TETH"
+		else if(locate(I) in heitem)
+			output = "HE"
+		else if(locate(I) in wawitem)
+			output = "WAW"
+		else if((locate(I) in alephitem) || (locate(I) in superEGO))
+			output = "ALEPH"
 		else
-			to_chat(user, "<span class='userdanger'>The well rejects your item!</span>")
+			output = "ZAYIN" //If an EGO is not in the lists for whatever reason it will default to zayin
 	else
-		to_chat(user, "<span class='userdanger'>The item flies back into your hand!</span>")
+		to_chat(user, "<span class='userdanger'>The well rejects your item!</span>")
+
+	// Now for outputs
+	if(!output)
+		return
+
+	if(!do_after(user, 15 SECONDS))
+		to_chat(user, "<span class='userdanger'>The well goes silent as it detects your impatience.</span>")
+		return
 
-//now for outputs
-	if(output)
-		gift = null
-		qdel(I)
-		gacha = rand(1,100)
-		if (gacha > 50)
-			output = null
-			if(do_after(user, 15 SECONDS))
-				playsound(src, 'sound/abnormalities//dreamingcurrent/dead.ogg', 80, TRUE, -3)
-				to_chat(user, "<span class='notice'>Nothing happens...</span>")
-			return
-		switch(output)
-			if ("MONEY")
-				switch(gacha)
-					if(1 to 5) //5% odds, spawn trash....
-						gift = pick(trash)
-					if(6 to 15)// 10% odds, down 1 risk level
-						gift = pick(baditem)
-					if(16 to 50)//35% odds, spawn an item of the same tier
-						gift = pick(normalitem)
-			if ("ZAYIN")
-				switch(gacha)
-					if(1 to 5)//5% odds, ...or a hostile mob if using EGO
-						gift = pick(pick(dawn),pick(trash))
-					if(6 to 15)
-						gift = pick(normalitem)
-					if(16 to 50)
-						gift = pick(zayinitem)
-			if ("TETH")
-				switch(gacha)
-					if(1 to 5)
-						gift = pick(dawn)
-					if(6 to 15)
-						gift = pick(zayinitem)
-					if(16 to 50)
-						gift = pick(tethitem)
-			if ("HE")
-				switch(gacha)
-					if(1 to 5)
-						gift = pick(noon)
-					if(6 to 15)
-						gift = pick(tethitem)
-					if(16 to 50)
-						gift = pick(heitem)
-			if ("WAW")
-				switch(gacha)
-					if(1 to 5)
-						gift = pick(dusk)
-					if(6 to 15)
-						gift = pick(heitem)
-					if(16 to 20)//5% odds to get a rarer item at WAW/ALEPH
-						gift = pick(/obj/item/ego_weapon/rabbit_blade,/obj/item/clothing/suit/armor/ego_gear/rabbit)
-					if(21 to 50)
-						gift = pick(wawitem)
-			if ("ALEPH")
-				switch(gacha)
-					if(1 to 5)
-						gift = pick(midnight)
-					if(6 to 15)
-						gift = pick(wawitem)
-					if(16 to 20)
-						gift = pick(pick(alephitem),/obj/item/toy/plush/bongbong) //egor says no PL/flowering/twilight
-					if(21 to 50)
-						gift = pick(alephitem)
+	var/gift = null
+	qdel(I)
+	var/gacha = rand(1,100)
+	if(gacha > 50)
+		playsound(src, 'sound/abnormalities//dreamingcurrent/dead.ogg', 80, TRUE, -3)
+		to_chat(user, "<span class='notice'>Nothing happens...</span>")
+		return
 
-//Gacha now locked in
+	switch(output)
+		if ("MONEY")
```

Comment:
According to coding standards, remove the space between the if-statement and it's condition, please.

Reply:
Wishing well needs a whole damn rewrite that I don't want to do at the moment, but sure, I can do that.